### PR TITLE
Cosmos: Bulk processing - reduce log noise level

### PR DESCRIPTION
**Pull request by FabianMeiswinkel**

### CODE REVIEW

**Review by ealsur — APPROVED** [review]

**Review by kushagraThapar — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java [hunk 1]
```diff
@@ -866,7 +866,18 @@ private void completeAllSinks() {
...
         } else {
-            logger.info("Main sink completion failed. EmitResult: {}, Context: {}", completeEmitResult, this.operationContextText);
+            if (completeEmitResult == Sinks.EmitResult.FAIL_CANCELLED ||
+                completeEmitResult == Sinks.EmitResult.FAIL_TERMINATED) {
+
+                logger.debug("Main sink already completed, EmitResult: {}, Context: {}",
+                    completeEmitResult,
+                    this.operationContextText);
+            } else {
+                logger.info(
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java:876]: Should this be info? I assume this is not a success state, so why not `warn` or `error` (if it is actionable) ?

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java [hunk 2]
```diff
@@ -866,7 +866,18 @@ private void completeAllSinks() {
         if (completeEmitResult == Sinks.EmitResult.OK) {
             logger.debug("Main sink completed, Context: {}", this.operationContextText);
         } else {
-            logger.info("Main sink completion failed. EmitResult: {}, Context: {}", completeEmitResult, this.operationContextText);
+            if (completeEmitResult == Sinks.EmitResult.FAIL_CANCELLED ||
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java:869]: Not part of this PR, but just curious to see that logger on line 862 - is info. Should we also change it to debug?

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java:869]: No - 862 being info is helpful - because it helps debugging in scenarios where some ingestion tasks take a long time (hang vs. long running task)

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java (hunk 1 above)
Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java:876]: I think with moving the high volumn logs to debug we can make this warning
